apps/server/src/modules/task/domain/TaskDomainService.spec.ts

```diff
@@ -1,165 +1,167 @@
-	});
-
-	describe('computeStatusForStudents', () => {
-		it('should compute status for each task', () => {
-			const { tasks, parents, helper } = prepareBaseData();
-			const domain = new TaskDomainService(tasks, parents);
-
-			const submission1 = helper.createSubmission(tasks[0]);
-			const submission2 = helper.createSubmission(tasks[1]);
-			const submission3 = helper.createSubmission(tasks[2]);
-			// 4 has no submission
-
-			const result = domain.computeStatusForStudents([submission1, submission2, submission3]);
-
-			const expected = {
-				graded: 0,
-				maxSubmissions: 1,
-				submitted: 1,
-			};
-
-			expect(result[0].status).toEqual(expected);
-			expect(result[1].status).toEqual(expected);
-			expect(result[2].status).toEqual(expected);
-			expect(result[3].status).toEqual({
-				graded: 0,
-				maxSubmissions: 1,
-				submitted: 0,
-			});
-		});
-
-		it('should always set maxSubmissions to one', () => {
-			const { tasks, parents, helper } = prepareBaseData();
-			const domain = new TaskDomainService(tasks, parents);
-			helper.createAndAddUser();
-
-			const submissions = helper.createSubmissionsForEachStudent(tasks[0]);
-
-			const result = domain.computeStatusForStudents(submissions);
-
-			const expected = {
-				graded: 0,
-				maxSubmissions: 1,
-				submitted: 2,
-			};
-
-			expect(result[0].status).toEqual(expected);
-		});
-	});
-
-	describe('computeStatusForTeachers', () => {
-		it('should set maxSubmissions to 0 if no parent is passed.', () => {
-			const { tasks, helper } = prepareBaseData();
-			const domain = new TaskDomainService(tasks, []);
-			helper.createAndAddUser();
-
-			const submissions = helper.createSubmissionsForEachStudent(tasks[0]);
-
-			const result = domain.computeStatusForTeachers(submissions);
-
-			const expected = {
-				graded: 0,
-				maxSubmissions: 0,
-				submitted: 2,
-			};
-
-			expect(result[0].status).toEqual(expected);
-		});
-
-		it('should compute status for each task', () => {
-			const { tasks, parents, helper } = prepareBaseData();
-			const domain = new TaskDomainService(tasks, parents);
-			helper.createAndAddUser();
-			helper.createAndAddUser();
-
-			const submissions1 = helper.createSubmissionsForEachStudent(tasks[0]);
-			const submissions2 = helper.createSubmissionsForEachStudent(tasks[1]);
-			const submission3 = helper.createSubmission(tasks[2]); // only for first user
-			// 4 has no submission
-
-			const result = domain.computeStatusForTeachers([...submissions1, ...submissions2, submission3]);
-
-			expect(result[0].status).toEqual({
-				graded: 0,
-				maxSubmissions: 10,
-				submitted: 3,
-			});
-			expect(result[1].status).toEqual({
-				graded: 0,
-				maxSubmissions: 10,
-				submitted: 3,
-			});
-			expect(result[2].status).toEqual({
-				graded: 0,
-				maxSubmissions: 10,
-				submitted: 1,
-			});
-			expect(result[3].status).toEqual({
-				graded: 0,
-				maxSubmissions: 10,
-				submitted: 0,
-			});
-		});
-	});
-});
+it.todo('moved to use case');

```

Comment:
wht's with the tests here? are they added somewhere else? then remove... otherwise: fix ;)

Reply:
the tests for the use case logic moved to task.uc.spec.ts. the domain service is not used anymore. can be deleted imho.